test/e2e/README.md

```diff
@@ -112,6 +112,15 @@ By default the description and logs for Antrea Pods are only written to disk if
 test fails. You can choose to dump this information unconditionally with
 `--logs-export-on-success`.
 
+### Testing the Prometheus Integration
+The Prometheus integration runs as a part of the e2e tests but should be enabled
+explicitly.
```

Comment:
Not sure that this is very clear. Maybe: "The Prometheus integration tests can be run as part of the e2e tests when enabled explicitly."?

Reply:
Yup better.